tools/setup-tools.sh

```diff
@@ -7,15 +7,15 @@
 # Name of file containing variable values
 template_fname=tools.env
 if [ ! -f "${template_fname}" ]; then
-  echo "Template file not found: ${template_fname}"
-  exit 1
+echo "Template file not found: ${template_fname}"
```

Comment:
Any reason for removing the whitespace here and a few lines below?

Reply:
No other than my editor kicking it violently :)